Civi/Api4/Action/Payment/Create.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+
+namespace Civi\Api4\Action\Payment;
+
+use Civi\Api4\Generic\Result;
+use Civi\Api4\CustomField;
+
+/**
+ * This API Action creates a payment. It is based on API3 Payment.create and API3 MJWPayment.create
+ *
+ */
+class Create extends \Civi\Api4\Generic\AbstractCreateAction {
+
+  /**
+   * If TRUE, format money fields to "standard format" eg. 1,00 => 1.00. Ideally pass in values in the correct format!
+   *
+   * @var bool
+   */
+  protected bool $cleanMoney = FALSE;
+
+  public function setCleanMoney(bool $value) {
```

Comment:
@mattwire I commented on this elsewhere - my instinct is not to support this & always set to false - we can add it later if we really decide there is a use case but can't remove later

Reply:
Right, I wasn't sure but agree it's nice not to have. Removed.